torch/ao/quantization/backend_config/executorch.py

```diff
@@ -346,6 +346,7 @@ def _get_share_qparams_ops_configs() -> List[BackendPatternConfig]:
     ]
     share_qparams_ops = [
         F.adaptive_avg_pool2d,
+        F.elu,
```

Comment:
also a bit strange that here F.leaky_relu_ is with inplace variant and F.relu/F.elu is without